### PR TITLE
Juhlee/doc add azure test

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
#68 
Can not add @lizzha in Reviewers. This is about a new paragraph how to add Azure test case. Please validate the instruction and example for the first time user.

### CODE REVIEW

File: Documents/How-to-use.md [hunk 1]
```diff
@@ -113,6 +116,25 @@ This XML file defines the regions per Category. It may require specific region o
...
+    1. Design test case and its configuration.
+    2. Create a new test case xml file under ./XML/TestCases folder. Or, update with new tags 
+        in the existing xml file.
+    3. Define testName, PowershellScript, setupType, Platform, Category, Area and Tags as required. 
+        Add optional tag if needed.
+    4. Test design may have two ways;
+        a. A single PowerShell script execution: A single PowerShell script imports builtin library 
+            modules and posts the result. For example, 'BVT-VERIFY-DEPLOYMET-PROVISION.ps1' shows 
+            this script calls 'DeployVMS' function for its testing and collect the result 
+            by 'CheckKernelLogs'. 'DeplyVM' and 'CheckKernelLogs' are builtin modules
```
[the remark this answers is not in the document]
Comment (lizzha) [reply Documents/How-to-use.md]: suggest to say: ‘DeployVMs’ and 'CheckKernelLogs' are functions defined in ./Libraries/CommonFunctions.psm1 module

File: Documents/How-to-use.md [hunk 2]
```diff
@@ -113,6 +116,25 @@ This XML file defines the regions per Category. It may require specific region o
...
+    4. Test design may have two ways;
+        a. A single PowerShell script execution: A single PowerShell script imports builtin library 
+            modules and posts the result. For example, 'BVT-VERIFY-DEPLOYMET-PROVISION.ps1' shows 
+            this script calls 'DeployVMS' function for its testing and collect the result 
+            by 'CheckKernelLogs'. 'DeplyVM' and 'CheckKernelLogs' are builtin modules 
+            in ./Libraries/CommonFunctions.psm1 module. 
+            You can add new module or update existing ones for further development.
+        b. PowerShell script wraps multiple non-PowerShell script like Bash or Python scripts: 
+            Like 'VERIFY-TEST-SCRIPT-IN-LINUX-GUEST.ps1', the PowerShell script wraps the multiple 
+            Bash or Python script as a parameter of 'RunLinuxCmd' PS module.
```
Comment (lizzha) [inline Documents/How-to-use.md]: 'RunLinuxCmd' function

File: Documents/How-to-use.md [hunk 3]
```diff
@@ -113,6 +116,25 @@ This XML file defines the regions per Category. It may require specific region o
...
+        Add optional tag if needed.
+    4. Test design may have two ways;
+        a. A single PowerShell script execution: A single PowerShell script imports builtin library 
+            modules and posts the result. For example, 'BVT-VERIFY-DEPLOYMET-PROVISION.ps1' shows 
+            this script calls 'DeployVMS' function for its testing and collect the result 
+            by 'CheckKernelLogs'. 'DeplyVM' and 'CheckKernelLogs' are builtin modules 
+            in ./Libraries/CommonFunctions.psm1 module. 
+            You can add new module or update existing ones for further development.
+        b. PowerShell script wraps multiple non-PowerShell script like Bash or Python scripts: 
+            Like 'VERIFY-TEST-SCRIPT-IN-LINUX-GUEST.ps1', the PowerShell script wraps the multiple
```
Comment (lizzha) [inline Documents/How-to-use.md]: suggest to use a real script name as example, cannot find 'VERIFY-TEST-SCRIPT-IN-LINUX-GUEST.ps1'

Comment (juhlee-microsoft) [reply Documents/How-to-use.md]: Please pull out the latest in repo. We have https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/VERIFY-TEST-SCRIPT-IN-LINUX-GUEST.ps1

File: Documents/How-to-use.md (hunk 2 above)
Comment (juhlee-microsoft) [reply Documents/How-to-use.md]: fixed

File: Documents/How-to-use.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (juhlee-microsoft) [reply Documents/How-to-use.md]: fixed

File: Documents/How-to-use.md [hunk 4]
```diff
@@ -113,6 +116,25 @@ This XML file defines the regions per Category. It may require specific region o
...
+## Add test case in Azure
+
+    1. Design test case and its configuration.
+    2. Create a new test case xml file under ./XML/TestCases folder. Or, update with new tags 
+        in the existing xml file.
+    3. Define testName, PowershellScript, setupType, Platform, Category, Area and Tags as required. 
+        Add optional tag if needed.
+    4. Test design may have two ways;
+        a. A single PowerShell script execution: A single PowerShell script imports builtin library 
+            modules and posts the result. For example, 'BVT-VERIFY-DEPLOYMET-PROVISION.ps1' shows
```
[the remark this answers is not in the document]
Comment (juhlee-microsoft) [reply Documents/How-to-use.md]: fixed

File: Documents/How-to-use.md [hunk 5]
```diff
@@ -113,6 +116,25 @@ This XML file defines the regions per Category. It may require specific region o
...
+    1. Design test case and its configuration.
+    2. Create a new test case xml file under ./XML/TestCases folder. Or, update with new tags 
+        in the existing xml file.
+    3. Define testName, PowershellScript, setupType, Platform, Category, Area and Tags as required. 
+        Add optional tag if needed.
+    4. Test design may have two ways;
+        a. A single PowerShell script execution: A single PowerShell script imports builtin library 
+            modules and posts the result. For example, 'BVT-VERIFY-DEPLOYMENT-PROVISION.ps1' shows 
+            this script calls 'DeployVMs' function for its testing and collect the result 
+            by 'CheckKernelLogs'. 'DeplyVMs' and 'CheckKernelLogs' are functions definded
```
Comment (iamshital) [inline Documents/How-to-use.md]: DeplyVMs --> DeployVMs

File: Documents/How-to-use.md [hunk 6]
```diff
@@ -113,6 +116,25 @@ This XML file defines the regions per Category. It may require specific region o
 
 Per Category, each XML file has VM name, Resource Group name, etc. We do not recommend to make change of the file.
 
+## Add test case in Azure
+
+    1. Design test case and its configuration.
+    2. Create a new test case xml file under ./XML/TestCases folder. Or, update with new tags 
+        in the existing xml file.
+    3. Define testName, PowershellScript, setupType, Platform, Category, Area and Tags as required.
```
Comment (iamshital) [inline Documents/How-to-use.md]: Tag is optional.

File: Documents/How-to-use.md (hunk 5 above)
Comment (juhlee-microsoft) [reply Documents/How-to-use.md]: fixed

File: Documents/How-to-use.md (hunk 6 above)
Comment (juhlee-microsoft) [reply Documents/How-to-use.md]: fixed

**Pull request merged**